clients/src/test/java/org/apache/kafka/common/network/SelectorTest.java

```diff
@@ -675,4 +712,11 @@ else if (obj instanceof Map)
             assertTrue("Field not empty: " + field + " " + obj, ((Map<?, ?>) obj).isEmpty());
     }
 
+    private KafkaMetric getMetric(String name) throws Exception {
+        for (Map.Entry<MetricName, KafkaMetric> entry : metrics.metrics().entrySet()) {
+            if (entry.getKey().name().equals(name))
+                return entry.getValue();
```

Comment:
Now that we can use Java 8 APIs, you can use `entrySet().stream().filter(...)` to avoid the for loop.